src/Microsoft.ML.TimeSeries/RootCauseAnalyzer.cs

```diff
@@ -40,7 +40,7 @@ private RootCause AnalyzeOneLayer(RootCauseLocalizationInput src)
             dst.Items = new List<RootCauseItem>();
 
             DimensionInfo dimensionInfo = SeparateDimension(src.AnomalyDimension, src.AggregateSymbol);
-            Tuple<PointTree, PointTree, Dictionary<Dictionary<string, object>, TimeSeriesPoint>> pointInfo = GetPointsInfo(src, dimensionInfo);
+            Tuple<PointTree, PointTree, Dictionary<Dictionary<string, Object>, TimeSeriesPoint>> pointInfo = GetPointsInfo(src, dimensionInfo);
```

Comment:
>Object [](start = 70, length = 6)

I would like suggest to change back to object. We should always use object instead of Object in our codebase, code clean up (change all the Object to object, Int to int etc...) is kind of beyong the scope of this PR.

Reply:
revert the coding style change